jobs/build/scan-images/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+final MAIL_FROM = 'aos-cd@redhat.com'
+final MAILING_LIST_CVE = ['bbarcaro@redhat.com']
+final MAILING_LIST_ERR = ['bbarcaro@redhat.com']
```

Comment:
Who should receive the notification email?

Reply:
smunilla, jupierce, ahaile

Reply:
Noted, I'll add them after we test the job.